homeassistant/components/media_player/sonos.py

```diff
@@ -199,6 +199,7 @@ def __init__(self, hass, player):
         self.hass = hass
         self.volume_increment = 5
         self._player = player
+        self._name = None
```

Comment:
My suggestion is to change this to
```python
self._name = self._player.uid()
```
to avoid ```entity_id``` clash for multiple Sonos per setup.